lib/block-supports/layout.php

```diff
@@ -136,29 +141,35 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 			 * by custom css.
 			 */
 			if ( ! empty( $layout['justifyContent'] ) && array_key_exists( $layout['justifyContent'], $justify_content_options ) ) {
-				$style .= "$selector {";
-				$style .= "justify-content: {$justify_content_options[ $layout['justifyContent'] ]};";
-				$style .= '}';
+				$layout_styles[ $selector ] = array( 'justify-content' => $justify_content_options[ $layout['justifyContent'] ] );
```

Comment:
As above, this overwrites the gap rule.

Reply:
fixed

Reply:
reverted fix... I rushed into it and did it wrong